src/DependencyInjection/Kfz24QueueExtension.php

```diff
@@ -24,27 +24,27 @@ class Kfz24QueueExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        $config        = $this->processConfiguration($configuration, $configs);
 
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('services.xml');
 
         foreach ($config['clients'] as $name => $client) {
             $clientType = $client['type'];
 
-            $apiVersion = $container->getParameter(sprintf('kfz24.queue.%s.api_version', $clientType));
+            $apiVersion   = $container->getParameter(sprintf('kfz24.queue.%s.api_version', $clientType));
             $adapterClass = $container->getParameter(sprintf('kfz24.queue.%s.adapter.class', $clientType));
-            $clientClass = $container->getParameter(sprintf('kfz24.queue.%s.client.class', $clientType));
+            $clientClass  = $container->getParameter(sprintf('kfz24.queue.%s.client.class', $clientType));
 
             $adapterDefinition = new Definition($adapterClass, [
                 [
-                    'region' => $client['region'],
-                    'endpoint' => $client['endpoint'],
+                    'region'      => $client['region'],
+                    'endpoint'    => $client['endpoint'],
                     'credentials' => [
-                        'key' => $client['access_key'],
+                        'key'    => $client['access_key'],
                         'secret' => $client['secret_access_key']
                     ],
-                    'version' => $apiVersion
+                    'version'     => $apiVersion
```

Comment:
[optional]

I am against of aligning the array values this way. 

It's noise in the PR, with insignificant benefit. Each time the array key changes every value has to be re-aligned, which adds noise again to the diff. 

I leave the decision up to the team.

Reply:
There were already some alignment, and when I  formatted I broke that. I tried to match the settings but seems I checked too many boxes (phpstorm settings per project). I'll fix it before submitting for official review.